### DISCUSSION TITLE
Error :Failed to add policy job since the add condition is not satisfied while enabling columnar engine in alloyDB instance

### DISCUSSION THREAD

**Discussion by noopur100** [TPC-Council/HammerDB#422] [Q&A] [answered]
Hi Team,

I am getting below error when I try to enable columnar engine on alloyDB, any help on this.
Thanks in advance.

Error :Failed to add policy job since the add condition is not satisfied

query used to enable columnar engine is below :
SELECT google_columnar_engine_add_policy(
  'RECOMMEND_AND_POPULATE_COLUMNS',
  'EVERY', 1, 'HOURS'
);

**Comment by sm-shaw** [comment, marked as answer]
Hi,
HammerDB has not been tested against alloyDB - it appears to be PostgreSQL compatible, if it is fully compatible then HammerDB should work against it. 

Saying that, this questions appears to be unrelated to the HammerDB side and should be directed to alloyDB instead - there is an example here of something similar that may answer your question:

https://stackoverflow.com/questions/72771059/warning-failed-to-add-policy-job-since-the-add-condition-is-not-satisfied
